tests/test_helpers.py

```diff
@@ -153,8 +153,16 @@ def test_astropy_table_value(self):
     def test_is_votable(self):
         """Result of convert() to votable should be a votable"""
         res = utils.convert(self.votable,'votable')
-        self.assertIsInstance(res,astropy.io.votable.tree.Table)
 
+        # astropy changed stuff with the votable in version 6.0.0
+        # https://github.com/astropy/astropy/blob/main/CHANGES.rst#astropyiovotable-3
+        if astropy.__version__ >= '6.0.0':
```

Comment:
Hi ! Astropy dev here.
Note that comparing version strings directly doesn't work as you'd expect and is not recommended
```python
>>> '10.0.0' > '2.0.0'
False
```

Fortunately, `astropy`  provides a more robust way to implement this:
using
```python
from astropy.utils import minversion
ASTROPY_GE_6_0 = minversion("astropy", "6.0.0")
```
you can rewrite this conditional as
```suggestion
        if ASTROPY_GE_6_0:
```